app/src/main/res/values/strings.xml

```diff
@@ -73,6 +73,7 @@
     <string name="inloggin">LOGIN</string>
     <string name="enterpasscode">You need to enter a 6 digit alphanumeric passcode.</string>
     <string name="resetpinrequest">Your request for reset has been received. You will receive an text message with a PIN reset confirmation code within %1$s.</string>
+    <string name="resetpinalert">Do you want to reset your pin?</string>
```

Comment:
can you please provide the translated string

Reply:
Please change this text into:
> Do you really want to reset your PIN? For security reasons, that will take time and you cannot use Yona in the meanwhile.

For Dutch:
> Wil je echt je pincode resetten? Om veiligheidsredenen gaat daar tijd overheen en ondertussen kan je Yona een niet gebruiken.